pandas/tests/frame/test_indexing.py

```diff
@@ -2939,6 +2939,15 @@ def test_where_callable(self):
         tm.assert_frame_equal(result,
                               (df + 2).where((df + 2) > 8, (df + 2) + 10))
 
+    def test_where_empty_df_and_empty_cond_having_non_bool_dtypes(self):
```

Comment:
can you move this tests near other tests of where which test empty

Reply:
Moved close to test_where_none